paasta_tools/utils.py

```diff
@@ -1426,7 +1426,12 @@ def get_readable_files_in_glob(glob: str, path: str) -> List[str]:
 )
 
 
+@time_cache(ttl=60)
```

Comment:
Do you think this will mess up deployd? Because it has a inotify watcher on this thing, so when it gets and event and.. calls this function nothing will be different maybe?

Reply:
That's a good point, I'll have to think about how to work around that.